commands/uptime_checks.go

```diff
@@ -0,0 +1,205 @@
+/*
+Copyright 2023 The Doctl Authors All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package commands
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/digitalocean/doctl"
+	"github.com/digitalocean/doctl/commands/displayers"
+	"github.com/digitalocean/doctl/do"
+	"github.com/digitalocean/godo"
+	"github.com/spf13/cobra"
+)
+
+// UptimeCheck creates the UptimeCheck command
+func UptimeCheck() *Command {
+	cmd := &Command{
+		Command: &cobra.Command{
+			Use:   "uptime",
+			Short: "Display commands to manage uptime checks",
+			Long: `The sub-commands of ` + "`" + `doctl uptime` + "`" + ` manage your uptime checks.
+
+DigitalOcean Uptime Checks provide the ability to monitor your endpoints from around the world,
+and alert you when they're slow, unavailable, or SSL certificates are expiring.`,
+		},
```

Comment:
```suggestion
			GroupID: manageResourcesGroup,
		},
```

After https://github.com/digitalocean/doctl/pull/1387 landed, we need to remember to `GroupID` to get them categorized right. As is, it shows up under `Additional Commands`.

Also, should this maybe be nested under the existing `monitoring` command? That would mirror the control panel experience.